quaddtype/README.md

```diff
@@ -25,28 +25,18 @@ np.array([1,2,3], dtype=QuadPrecDType("longdouble"))
 
 ## Installation from source
 
-The code needs the quad precision pieces of the sleef library, which is not available on most systems by default, so we have to generate that first. Choose the appropriate section below based on your operating system.
-
-### Linux/Unix/macOS
+#### Prerequisites
 
-The below assumes one has the required pieces to build sleef (cmake and libmpfr-dev), and that one is in the package directory locally.
+- **gcc/clang**
+- **CMake** (≥3.15)
+- **Python 3.10+**
+- **Git**
 
-```bash
-git clone --branch 3.8 https://github.com/shibatch/sleef.git
-cd sleef
-cmake -S . -B build -DSLEEF_BUILD_QUAD:BOOL=ON -DSLEEF_BUILD_SHARED_LIBS:BOOL=ON -DCMAKE_POSITION_INDEPENDENT_CODE=ON
-cmake --build build/ --clean-first -j
-cd ..
-```
+### Linux/Unix/macOS
 
 Building the `numpy-quaddtype` package from locally installed sleef:
```

Comment:
Do we always assume that sleef is already locally installed? If not, now does the command differ if it isn't?